spec/spec_helper.rb

```diff
@@ -43,6 +43,15 @@
     DatabaseCleaner.strategy = :transaction
     DatabaseCleaner.clean
     Indexer.all.map(&:prepare_for_testing)
+    ArchiveWarning.find_or_create_by_name(ArchiveConfig.WARNING_CHAN_TAG_NAME).update(canonical: true)
```

Comment:
In an ideal world, we'd just create all the default tags in an initializer. This is not an ideal world.